reminders/class_tests/update_reminder_tests.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from reminders.forms import ReminderUpdateForm
+from pytest_django.asserts import assertTemplateUsed
+
+
+@pytest.mark.django_db
+class TestUpdateReminder:
+
+    @pytest.mark.parametrize("invalid_user_credentials", [
+        # method cannot be None
+        {'participant_id': None, 'date_time': pytest.valid_date_time,
+         'messages': pytest.message, 'method': None
+         },
+        # method can be only from ReminderType
+        {'participant_id': None, 'date_time': pytest.valid_date_time,
+         'messages': pytest.message, 'method': pytest.invalid_method
+         },
+        # datetime should be greater then the present time
+        {'participant_id': None, 'date_time': pytest.invalid_date_time,
+         'messages': pytest.message, 'method': pytest.valid_method
+         }
+    ], ids=[
+        "method is none",
+        "method is not from ReminderType",
+        "date is before the current time"
+        ]
+    )
+    def test_create_reminder(self, get_event_participant, invalid_user_credentials):
```

Comment:
please change name to invalid reminder

Reply:
done as you can see in the above comment